Source/SwiftLintFramework/Rules/FunctionParameterCountRule.swift

```diff
@@ -91,6 +101,14 @@ public struct FunctionParameterCountRule: ASTRule, ConfigurationProviderRule {
             .characters.filter { $0 == equalCharacter }.count ?? 0
     }
 
+    fileprivate func functionIsInitializer(_ file: File, offset: Int, length: Int) -> Bool {
+        if let function = (file.contents as NSString)
```

Comment:
this might matter when we add Linux support, but the implicit bridging works fine for now and is cleaner, so let's stick with the cleaner approach.